.drone.star

```diff
@@ -153,6 +153,7 @@ config = {
     },
     "rocketchat": {
         "channel": "infinitescale",
+        "channel_builds": "builds",
```

Comment:
```suggestion
        "channel_cron": "builds",
```

This is the channel to report to for drone "cron" jobs (i.e. when the build event is "cron").
So I suggest naming it that.